hypervisor/dm/vuart.c

```diff
@@ -559,18 +559,11 @@ static bool vuart_register_io_handler(struct acrn_vm *vm, uint16_t port_base, ui
 		.base = port_base,
 		.len = 8U
 	};
-	switch (vuart_idx) {
-	case 0U:
-		pio_idx = UART_PIO_IDX0;
-		break;
-	case 1U:
-		pio_idx = UART_PIO_IDX1;
-		break;
-	default:
+	if (vuart_idx >= MAX_VUART_NUM_PER_VM) {
```

Comment:
Can you handle normal case first?

Reply:
Sure, will update it soon, thanks!

Reply:
@dongyaozu Hi eddie, updates have been done, please double check it, thanks!